src/test/java/org/jenkinsci/plugins/scm_filter/GitHubPullRequestSkipTraitTest.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.scm_filter;
+
+import jenkins.model.Jenkins;
+import jenkins.scm.api.SCMSource;
+
+import org.jenkinsci.plugins.github_branch_source.GitHubSCMSource;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertThat;
+
+
+public class GitHubPullRequestSkipTraitTest {
+
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
+
+    @Rule
+    public TestName currentTestName = new TestName();
+
+
+    private SCMSource load () {
+        return load(currentTestName.getMethodName());
+    }
+
+
+    private SCMSource load (String dataSet) {
+        return (GitHubSCMSource) Jenkins.XSTREAM2.fromXML(
+                getClass().getResource(getClass().getSimpleName() + "/" + dataSet + ".xml")
```

Comment:
8 spaces for indent? ... well, I guess we don't have a convention for java ...

Reply:
That's what IntelliJ CTRL+ALT+L suggests, so I just followed it.